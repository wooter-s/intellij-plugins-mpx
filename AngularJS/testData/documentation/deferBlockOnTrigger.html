@defer (prefetch on view<caret>port(foo))