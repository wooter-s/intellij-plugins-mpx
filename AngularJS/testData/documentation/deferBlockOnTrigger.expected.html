<div class='definition'><pre><span style='color: #000080'>trigger</span> <span style='color: #000000'>viewport</span></pre></div>
<div class='content'>
<p><code>viewport</code> would trigger the deferred block when the specified content enters the viewport using the <code>IntersectionObserver</code> API. This could be the placeholder content or an element reference.</p><p>By default, the placeholder will act as the element watched for entering viewport as long as it is a single root element node. Alternatively, you can specify a template reference variable in the same template as the <code>@defer</code> block as the element that is watched to enter the viewport.</p>
</div><table class='sections'>
<tr><td valign='top' class='section'><p>Library:</td><td valign='top'><p>@angular/core@17.0.0-rc.0</td>
</table>